toolbackup/src/test/java/io/dockstore/tooltester/client/cli/ClientTest.java

```diff
@@ -0,0 +1,17 @@
+package io.dockstore.tooltester.client.cli;
+
+import joptsimple.OptionParser;
+import joptsimple.OptionSet;
+import org.junit.Test;
+
+public class ClientTest {
+    @Test
+    public void setupEnvironment() throws Exception {
+        OptionParser parser = new OptionParser();
+        final OptionSet parsed = parser.parse("");
+        Client client = new Client(parsed);
+        client.setupClientEnvironment();
+        //Assert.assertTrue("client API could not start", client.getContainersApi() != null);
```

Comment:
Looks good so far. I think your next tasks after completing output for multiple runs will include
1) Write tests run them using Travis-CI (ideas for tests include: test reading and saving a small docker image (maybe hello-world), standing up S3Ninja and then writing data to it, validate the output HTML with either a validation service or a Java library, etc.)
2) Hook up coverage testing so we know how well your code is tested
3) Document how to use your script (and read its output)